workflow/doc.go

```diff
@@ -228,10 +228,10 @@ for an activity it invoked.
 		WorkflowID:                   "BID-SIMPLE-CHILD-WORKFLOW",
 		WorkflowExecutionTimeout: time.Minute * 30,
 	}
-	ctx = workflow.WithChildOptions(ctx, cwo)
+	childCtx = workflow.WithChildOptions(ctx, cwo)
```

Comment:
I don't think this change is needed. it is absolutely ok to reuse the same variable.